test/powershell/engine/Basic/StandardLibraryTypes.Tests.ps1

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@JamesWTruher, your last commit had 709 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Types referenced by PowerShell Standard should not be missing.Type 'System.Management.Automation.ExecutionContext' should be present with correct attributes
```powershell
IsCollectible value is not correct
at <ScriptBlock>, /Users/runner/runners/2.164.6/work/1/s/test/powershell/engine/Basic/StandardLibraryTypes.Tests.ps1: line 30
30:                 throw "$property value is not correct"

```
Types referenced by PowerShell Standard should not be missing.Type 'System.Management.Automation.OutputProcessingStateEventArgs' should be present with correct attributes
```powershell
IsCollectible value is not correct
at <ScriptBlock>, /Users/runner/runners/2.164.6/work/1/s/test/powershell/engine/Basic/StandardLibraryTypes.Tests.ps1: line 30
30:                 throw "$property value is not correct"

```
Types referenced by PowerShell Standard should not be missing.Type 'System.Management.Automation.PSRemotingJob' should be present with correct attributes
```powershell
IsCollectible value is not correct
at <ScriptBlock>, /Users/runner/runners/2.164.6/work/1/s/test/powershell/engine/Basic/StandardLibraryTypes.Tests.ps1: line 30
30:                 throw "$property value is not correct"

```
Types referenced by PowerShell Standard should not be missing.Type 'System.Management.Automation.ActionPreference' should be present with correct attributes
```powershell
IsCollectible value is not correct
at <ScriptBlock>, /Users/runner/runners/2.164.6/work/1/s/test/powershell/engine/Basic/StandardLibraryTypes.Tests.ps1: line 30
30:                 throw "$property value is not correct"

```
Types referenced by PowerShell Standard should not be missing.Type 'System.Management.Automation.ActionPreferenceStopException' should be present with correct attributes
```powershell
IsCollectible value is not correct
at <ScriptBlock>, /Users/runner/runners/2.164.6/work/1/s/test/powershell/engine/Basic/StandardLibraryTypes.Tests.ps1: line 30
30:                 throw "$property value is not correct"

```